NebulaModel/Packets/Logistics/ILSSystemStorageSync.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NebulaModel.Packets.Logistics
+{
+    public class ILSSystemStorageSync
+    {
+        public int[] stationGId { get; set; }
+        public int[] storageStationGId { get; set; }
```

Comment:
No constructor with array copy here ?

Reply:
Ah yes, i started to work on something but made an individual issue out of it. Im gonna remove this file for now.